drools-wb-webapp/src/main/java/org/drools/workbench/backend/server/LuceneConfigProducer.java

```diff
@@ -40,8 +40,6 @@
 import org.uberfire.ext.metadata.backend.lucene.LuceneConfigBuilder;
 import org.uberfire.ext.metadata.backend.lucene.analyzer.FilenameAnalyzer;
 
-import static org.apache.lucene.util.Version.*;
-
 @ApplicationScoped
 public class LuceneConfigProducer {
```

Comment:
@manstis Hi, I think it could. However, since it is a backend class and this issue only covers client-side code, I will open another JIRA issue to take care of that, wdyt?